ocs/defaults.py

```diff
@@ -17,57 +13,14 @@
 OCP_API_VERSION = 'project.openshift.io/v1'
 OPENSHIFT_REST_CLIENT_API_VERSION = 'v1'
 
-# Those variables below are duplicate at the moment from default_config.yaml
-# and once we drop support for old runner we will remove those variables from
-# here and will have them defined only on one place.
-
 # Be aware that variables defined above and below are not used anywhere in the
 # config files and their sections when we rendering config!
 
 INSTALLER_VERSION = '4.1.2'
 CLIENT_VERSION = INSTALLER_VERSION
-AWS_REGION = 'us-east-2'
 ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
 KUBECONFIG_LOCATION = 'auth/kubeconfig'  # relative from cluster_dir
-CLUSTER_NAME = "ocs-ci"
 API_VERSION = "v1"
-CEPH_IMAGE = "ceph/ceph:v14"
-ROOK_IMAGE = "rook/ceph:master"
-DEPLOYMENT_PLATFORM = 'AWS'
-
-# This section is suppose to be available just from ocsci/config.py module from
-# ENV_DATA dictionary. Once we drop support of old runner we will delete this
-# data from here as well.
-ENV_DATA = {
-    'platform': DEPLOYMENT_PLATFORM,
-    'cluster_name': CLUSTER_NAME,
-    'cluster_namespace': ROOK_CLUSTER_NAMESPACE,
-    'region': AWS_REGION,
-    'ceph_image': CEPH_IMAGE,
-    'rook_image': ROOK_IMAGE,
-}
-
-DEPLOYMENT = {
-    'installer_version': INSTALLER_VERSION,
-}
-
-REPORTING = {
-    'email': {
-        'address': 'ocs-ci@redhat.com',
-    },
-    'polarion': {
-        'project_id': 'OpenShiftContainerStorage',
-    }
-}
-
-RUN = {
-    'log_dir': '/tmp',
-    'run_id': None,
-    'kubeconfig_location': 'auth/kubeconfig',
-    'cli_params': {},
-    'client_version': DEPLOYMENT['installer_version'],
-    'bin_dir': './bin',
-}
```

Comment:
If you read the comments on the top of the ENV_DATA, it was never supposed to be here in the first place. I am still not clear what *specific* changes you are talking about,  I am removing the data that is not used by any file but exists in defaults.py and removing default values that just exists and used by ENV_DATA

Reply:
@zmc once this will be merged I will submit other PR or you can finish in your to remove completely the defaut_config.yaml and we can have those vales here and feed the default config.

So not sure now if it's worth to remove them from here if we will return those values here again very soon.  But we can just copy paste from old commit so I don't mind. But yep those values are not used anywhere now.

Reply:
@vasukulkarni specifically, I disagree with removing `ENV_DATA`, `DEPLOYMENT`, `REPORTING` and `RUN` here.

If the idea is to eventually have all these values in `defaults.py`, I don't see the point in removing them here. It seems counter-productive. I'd rather see them removed from `default_config.yaml`, as the goal is to remove that file entirely.